externals/kyuubi-chat-engine/src/main/scala/org/apache/kyuubi/engine/chat/provider/ChatGPTProvider.scala

```diff
@@ -32,9 +33,21 @@ import org.apache.kyuubi.engine.chat.provider.ChatProvider.mapper
 
 class ChatGPTProvider(conf: KyuubiConf) extends ChatProvider {
 
-  val token = conf.get(KyuubiConf.ENGINE_CHAT_GPT_API_KEY).get
+  private val token = conf.get(KyuubiConf.ENGINE_CHAT_GPT_API_KEY).get
```

Comment:
TODO(in future PR): improvement the error message if user does not provide the api key.